framework/src/testing/strategy.rs

```diff
@@ -0,0 +1,54 @@
+//! proptest strategies
+
+use crate::packets::ip::{Flow, ProtocolNumbers};
+use proptest::arbitrary::any;
+use proptest::strategy::{Just, Strategy};
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+
+/// Returns a strategy to generate IPv4 flows
+pub fn v4_flow() -> impl Strategy<Value = Flow> {
```

Comment:
may be worth simple examples in the comments, just to show how you'd use these in an nf? @drunkirishcoder just airing that out, but if you feel it's unnecessary, np.

Reply:
I will add a little more text on what's being generated from this. but usage wise, it's pretty much just `proptest` since this doesn't take any input customization.